samples/Sentry.Samples.Maui/App.xaml.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Sentry.Samples.Maui;
+
+public partial class App : Application
+{
+	public App()
+	{
+		InitializeComponent();
```

Comment:
Tabs 😭

Reply:
Came with the sample app template, lol.  Will update.